### ISSUE TITLE
persist JSON word-coordinate from ingested NewspaperPage image

### ISSUE THREAD

**Issue by ebenenglish** [marriott-library/newspaper_works#44]
**Descriptive summary**
In addition to ALTO, we want to persist a JSON serialization of the word-coordinate data. This file will allow us to quickly serve word-coordinate data to drive front-end behavior, for viewing/searching/highlighting, generating an IIIF Annotation List, etc. 

**Background**
We will use an [Open-ONI-type](https://intranet.lib.utah.edu/download/attachments/25853987/open-oni_OCR-word-coordinates.json?api=v2) structure for the JSON data. However, we will 
_not_ normalize (i.e. combine multiple entries of the same word) the data, which will allow us to preserve word order.

Sample structure:
```json
[
  { "word": "If",  "coordinates": [52, 13, 11, 14] },
  { "word": "you",  "coordinates": [69, 17, 31, 14] },
  { "word": "are",  "coordinates": [108, 17, 28, 10] }
]
```

**Expected behavior**
As an admin user
When I upload a page object
The application should create a JSON file representing the word-coordinate data with the desired structure
And it should persist the JSON file in the same fashion as other derivatives

**Comment by seanupton** [comment]
Work for this is being done on `https://github.com/marriott-library/newspaper_works/tree/page-wordjson` branch.

**Comment by seanupton** [comment]
@ebenenglish the format used at present is to have coordinates as named members of a hash, not unnamed members of an array with a presumed order. 

```json
{
  "words" : [
    {"word": "If", "x_start": 33, "y_start": 10, "width": 18, "height":15},
    {"word": "you", "x_start": 56, "y_start": 10, "width": 28, "height":15}
  ]
}
```

Thoughts?

**Comment by ebenenglish** [comment]
I'm open to discussion, but my preference is more in the direction of "unnamed members of an array with a presumed order."

Reasons being:
* This derivative is about supporting front-end behavior, and so the more the data is pre-prepped, the better.
* The presumed order seems to be a reasonably well-understood convention within the community (Library of Congress, open-oni, IIIF).
* If someone really needs the individual keys broken out, the ALTO is always available.

Again, willing to listen to arguments for named-members-of-a-hash approach.

**Comment by seanupton** [comment]
I'm fine with that, and will assume the order is `[x_start, y_start, width, height]`.  This will be more compact, which I think is good (esp. considering we are not compressing this stuff).  I will adjust this.

**Comment by ebenenglish** [comment]
Closed via #48.
